include/string.h

```diff
@@ -91,8 +120,19 @@ int strverscmp (const char *, const char *);
 char *strchrnul(const char *, int);
 char *strcasestr(const char *, const char *);
 void *memmem(const void *, size_t, const void *, size_t);
```

Comment:
Please add a comment here why we did not convert the `memmem` function to Checked C. Also please add a reference to the issue https://github.com/microsoft/checkedc-clang/issues/868 here.